src/content/staking/pools/index.md

```diff
@@ -57,17 +57,22 @@ Please note the importance of choosing a service that takes <a href="/developers
 
 Have a suggestion for a staking tool we missed? Check out our [product listing policy](/contributing/adding-staking-products/) to see if it would be a good fit, and to submit it for review.
 
-## FAQ {#faq}
+## Frequently asked questions {#faq}
 
 <ExpandableCard title="How do I earn rewards?">
 Typically ERC-20 liquidity tokens are issued to stakers that represents the value of their staked ETH plus rewards. Keep in mind that different pools will distribute staking rewards to their users via slightly different methods, but this is the common theme.
 </ExpandableCard>
 
 <ExpandableCard title="When can I withdraw my stake?">
+Currently, withdrawing funds from an Ethereum validator is not possible, which limits the ability to actually <i>redeem</i> your liquidity token for the ETH rewards locked in the consensus layer.
 
-Currently, withdrawing funds from an Ethereum validator is not possible, which limits the ability to actually _redeem_ your liquidity token for the ETH rewards locked in the consensus layer.
+The Shanghai network upgrade will introduce withdrawal functionality which is planned for Q1/Q2 2023. All staked ETH will remain locked until then.
+
+After this upgrade, the validator accounts that back staking pool will have the ability to exit and withdrawal ETH to their designated withdrawal address. This will enable the ability to redeem your portion of stake for the underlying ETH. Check with your provider to see how they support this functionality.
```

Comment:
```suggestion
After this upgrade, the validator accounts that back staking pools will have the ability to exit and withdraw ETH to their designated withdrawal address. This will enable the ability to redeem your portion of stake for the underlying ETH. Check with your provider to see how they support this functionality.
```

I think this makes a bit more sense.